zowe-cli-id-federation-plugin/npm-shrinkwrap.json

```diff
@@ -62,12 +63,15 @@
       }
     },
     "node_modules/@babel/code-frame": {
-      "version": "7.12.11",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz",
-      "integrity": "sha512-Zt1yodBx1UcyiePMSkWnU4hPqhwq7hGi2nFL1LeA3EUl+q2LQx16MISgJ0+z7dnmgvP9QtIleuETGOiOH1RcIw==",
+      "version": "7.18.6",
+      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.18.6.tgz",
```

Comment:
Shall we use this public URL or rather JFrog? ..like `https://zowe.jfrog.io/artifactory/npm-org-cache/%40babel/code-frame/-/code-frame-7.18.6.tgz`.

I guess `https://registry.npmjs.org` should be replaced by  `https://zowe.jfrog.io/artifactory/npm-org-cache`.